client/devdocs/design/syntax.scss

```diff
@@ -0,0 +1,89 @@
+code[class*="lang-"] {
+    color: #393A34;
+    direction: ltr;
+    text-align: left;
+    white-space: pre;
+    line-height: 1.2em;
+
+    -moz-tab-size: 4;
+    -o-tab-size: 4;
+    tab-size: 4;
+
+    -webkit-hyphens: none;
+    -moz-hyphens: none;
+    -ms-hyphens: none;
+    hyphens: none;
+}
+
+code[class*="lang-"]::selection, code[class*="lang-"] ::selection,
+code[class*="lang-"]::-moz-selection, code[class*="lang-"] ::-moz-selection {
+    background: #b3d4fc;
+}
+
+.token.comment,
```

Comment:
I do believe there is a plug-in that allows for a custom class, which might be worth adding to better namespace these styles. I'll add that to https://github.com/Automattic/wp-calypso/pull/10359#issuecomment-271124311 .